### ISSUE TITLE
Tool tip for schedule and order cycle on the OC admin page

### ISSUE THREAD

**Issue by sstead** [openfoodfoundation/openfoodnetwork#1394]
"In the Order Cycle admin page there is an Add Schedule button - clicking this opens a modal"... This button didn't appear on the page until Standing Orders was enabled in Enterprise > Manage. I only knew this was an issue due to Nick's comment.

As a User I am not familiar with the terminology - but ‘schedule’ and ‘order cycle’ were not easily understood. Consider adding a tooltip for the labels?

![image](https://cloud.githubusercontent.com/assets/10843056/22269562/0a3e3630-e2e0-11e6-8ff9-d06bca28f2d7.png)

* Issue spotted by Yuko

**Comment by oeoeaio** [comment]
Hey @yukoosawa. Sorry, could you provide little more clarification about what it is you'd like to see for this, and where? Ta!

**Comment by yukoosawa** [comment]
@oeoeaio So it may have just been a me thing (and it is very possible that you cover this information as part of onboarding a new customer/enterprise) but I wasn't sure what 'Schedule' and 'Order Cycle' meant in the buttons.

<img width="1004" alt="screen shot 2017-03-01 at 10 15 34 pm" src="https://cloud.githubusercontent.com/assets/19785399/23491613/56feefd8-fecd-11e6-91e1-5fc7ab60b115.png">

I wasn't entirely sure of the difference (although I know now) and wasn't sure of best practices especially when it came to naming and organising these.

Suggestion was to add a tooltip or something to help the user understand what the labels/actions mean. But again please note my comment was very much driven by being a complete noobie to admin/enterprise/shop/ofn language :) This is a bootstrap tooltip:
<img width="846" alt="screen shot 2017-03-01 at 10 17 07 pm" src="https://cloud.githubusercontent.com/assets/19785399/23491687/ca03e1d2-fecd-11e6-83ae-39ae924c6849.png">

**Comment by oeoeaio** [comment]
👍

**Comment by myriamboure** [comment]
@sstead can you specify then more clearly with template the issue with the result of what is expected from a dev so that it can be picked up? thanks ;-) Also it seems there need to  be some review on UX for where to put the schedule notion, apparently Rob was not super happy to have it under order cycle... so shall we pause that issue? As if we review that soon maybe it doens't worth fixing this... if you agree we can put as blocked for now. Else specify and let's ask some newby to work on it !:-)

**Comment by myriamboure** [comment]
So I did a bit of analysis on this issue to sort out the last two unclear issues on subs.

Today on OFN the bootstrap tooltip appear in two situations if I'm not mistaken:
- on pictos at the end of order lines
![capture du 2019-01-10 19-41-00](https://user-images.githubusercontent.com/8135376/50989555-f88a0b80-150f-11e9-9905-9ecae01afede.png)

- on entesprise settings under "what's this" text
![capture du 2019-01-10 19-41-18](https://user-images.githubusercontent.com/8135376/50989559-00e24680-1510-11e9-8ac8-8879b3c02108.png)

I checked all the other main menu pages, filters and call to actions (+ new xxx) and non has any tooltip. My intuition is that if the main CTA needs a tooltip there is a UX problem.
Order cycle notion is a fundamental notion on OFN and we explain it to any new user. I don't think it needs a tooltip, if they want to understand the logic they need to go to the user guide.

So my preference would be to:
- to clearly add a "user guide" button very visible that is accessible from any page of the back office (today only from enterprise menu). For instance it could be here
![untitled presentation](https://user-images.githubusercontent.com/8135376/50989908-efe60500-1510-11e9-91ab-1025ab2dd6ff.jpg)
- and when we have fixed the bugs and iterate on the subs feature, improve "schedule" UX, as Rob said it's not easily understandable to have it with Order Cycle. I had the same feedback from our beta tester. But I won't open an issue, just record that as one of the things we need to work on for subs v2.

What do you think about that suggestion ? @sstead @sstead @kirstenalarsen ?

**Comment by myriamboure** [comment]
:point_up: @kirstenalarsen @lin-d-hop @RachL woudl be great also to have your feedback on my proposal for this second and last story that was in "define and design" in sub, so I need some review before specifying what to do. Thanks !

**Comment by sauloperez** [comment]
Independent from subs, providing the link to the guide would definitely help newcomers get used to the product and let others learn about new features. Although using "help" might be a bit easier to understand?

**Comment by RachL** [comment]
@myriamboure As much as I would like to have a clear access to the user guide, I don't think this is something we can squeeze in the Subs v1 scope. This to me is another issue, even something we need to put to discourse first? What do you think @daniellemoorhead ?

Here we need:
1. To agree where we put the link
2. Make sure we remove the other buttons 
3. Make sure the work Hugo did so that each instance can dynamically add their own user link in the configurations menu still works

Regarding 1, I'm not sure it is a good practice to have links next to each other. The more we have the more it will show up weirdly with some resolutions. So maybe we need to introduce something like a profile (like we have in front office) ? So that the User guide link is a direct one but the the others are more in a sub menu?

Also I don't know if the orange you put is on purpose? I'm afraid the typo color we have in the backoffice does not give much contrast and can be hard for people with bad eyesight. So I would be in favor of a black text :)

**Comment by sigmundpetersen** [comment]
Re. tooltips I quite like the small question mark icon: 
![help](https://user-images.githubusercontent.com/1451633/52620866-11d2fe80-2ea6-11e9-827b-50190e1cbef6.png)

**Comment by sauloperez** [comment]
Agree with @RachL on decoupling these two things.

@sigmundpetersen I do too. It's rather common so I expect users to be familiar with it.

**Comment by daniellemoorhead** [comment]
>@myriamboure As much as I would like to have a clear access to the user guide, I don't think this is something we can squeeze in the Subs v1 scope. This to me is another issue, even something we need to put to discourse first? What do you think @daniellemoorhead ?

I agree, not necessarily part of subs. But it would make what we have great...so let's not lose this when it's taken out of the subs epic.

**Comment by sigmundpetersen** [comment]
I found that already have an example of the question mark style tooltip in the backoffice 😄 
![q](https://user-images.githubusercontent.com/1451633/52858333-1a3b6b80-312a-11e9-8f90-5ca71ac0ab39.png)

**Comment by luisramos0** [comment]
Removing from the subs v1 epic as it's not part of the scope.

**Comment by kirstenalarsen** [comment]
not scoped clearly enough for papercut

**Comment by kirstenalarsen** [comment]
@lin-d-hop @RachL do either of you think this is scoped clearly enough to be on the Welcome New Devs board? It isn't to me - I am removing but can put back if you think it is?

**Comment by Erioldoesdesign** [comment]
Question: Was there a decision made to call subscriptions 'subscriptions' in the settings of the enterprise:
<img width="517" alt="Screenshot 2020-11-02 at 15 21 53" src="https://user-images.githubusercontent.com/11681324/97885428-41cbd080-1d1f-11eb-9c1a-ac1d21f2934d.png">

But then refer to that new functionality as 'schedules':
<img width="158" alt="Screenshot 2020-11-02 at 15 21 59" src="https://user-images.githubusercontent.com/11681324/97885464-4db79280-1d1f-11eb-9e24-c0f6d4df5807.png">
<img width="744" alt="Screenshot 2020-11-02 at 15 22 03" src="https://user-images.githubusercontent.com/11681324/97885468-4f815600-1d1f-11eb-9939-c139a0346d0f.png">


I wonder if, the papercut sized solution for this is to replace 'schedule' with 'subscription'. Or at least reference subscriptions as a function in this model:
<img width="744" alt="Screenshot 2020-11-02 at 15 22 03" src="https://user-images.githubusercontent.com/11681324/97885571-72ac0580-1d1f-11eb-9191-4ea940fffcaf.png">


Maybe under the heading text  of 'adding a new schedule' we can add text like:

"This is where you create subscription like options for your customers. The subscription or 'schedule' must be attached to an order cycle" etc.

**Comment by RachL** [comment]
@Erioldoesdesign FYI all instances can change basically all text within the app thanks to translations. So for example in FR we call subscriptions "recurring orders" because we thought it was more appropriate.

That being said I'm not sure I understand your proposal: a schedule is like a tag which - when applied to an order cycle - triggers the subscription process.

But subs are not created when you create a schedule. They are created beforehand in `/admin/subscriptions`.

If you think it can be useful, maybe we could set up a call where I give you a little demo of how it works currently.

**Comment by Erioldoesdesign** [comment]
hmm Louise did some subscriptions training with me already but looks like it didn't quite stick in my mind.

a demo is always welcome! 

It looks like my suggestion for papercutting this again won't work. However the confusion between order cycles, subscriptions and schedules remains true then and it seems they are functions that will require user guide/customer support unless simplification is done. So that part is not papercut.

What Yuko originally suggested though was adding context through a tool tip as to why the 'New schedule' button has suddenly appeared after 'enabling' subscriptions. It is not at all clear what you would do next after enabling subscriptions (unless you are following the guide or a customer support person does this for you)

**Comment by luisramos0** [comment]
The only word I think could be better than Schedule would may be Rhythm of the subscription.
This is better understood with the basic example of "weekly" and "fortnightly". You have a weekly OC but you need those two schedules because some customers will only want to receive what's on their "subscription" every other week.
**A Subscription is not directly connected to an Order Cycle**, only through the Schedules. For example, customer A has a subscription with schedule fortnightly and customer B has a subscription with schedule weekly. Customer A will not have orders in half of the weekly Order Cycles.
I hope this helps explain the concepts.

I am happy we are talking about this. There's plenty of room for improvement in terms of making it easier to use/understand :tada:
